.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - ci-test
+  pull_request:
+
+jobs:
+  check-graphql-schema:
+    name: Check that frontend is built against our latest graphql schema
+    runs-on: [ubuntu-latest]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Start hasura
+        working-directory: ./
+        run: docker-compose up -d hasura
+
+      - name: Wait for hasura instance to get up
+        run: while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost:8080/healthz)" != "200" ]]; do echo "Waiting for hasura instance..."; sleep 5; done
+
+      - name: Fetch latest graphql schema
+        working-directory: ./frontend
+        run: npx graphqurl "http://localhost:8080/v1beta1/relay" --introspect > latest-schema.graphql
```

Comment:
En ole kovin tyytyväinen siihen, että tämä ja edellinen steppi ovat nyt pitkälti samaa kuin `frontend/scripts/import-graphql-schema.sh`, jota voi siis käyttää esim. lokaalisti devatessaan.
Seuraavat asiat pitää ratkoa jos haluaa käyttää samaa scriptiä täälläkin:
- HASURA_URL pitää uittaa jotenkin scriptille - esim. env-variablen kautta (tämä onkin jo tuettuna)
- output file location pitää uittaa jotenkin scriptille - esim. komentoriviparametrina tai env-muuttujana, mutta ensin mainittu vaatinee jonkun verran boilerplatea jos sen haluaa tehdä siististi ja jälkimmäinen taas on ehkä vähän purkaratkaisu
- kys. scripti käyttää node_modulesissa olevaa graphqurl-kirjastoa, jota ei tässä kontekstissa ole asennettuna. Tämä ratkeaisi toki sillä että käyttäisi npx:ää myös devatessa, mutta onko sekään kovin nättiä?

Reply:
En osaa näin maallikkona ottaa kantaa onko `npx`:n käyttö eleganttia vai ei. Ainakin mun silmään se näyttää paremmalta kuin kutsua jotain `node_modules/foo/bar/baz/dist/src/bin/foo/foo` binääriä "suoraan", mutta tässä lienee jotain teknisiä aspekteja, jotka ei mulle aukea.

Toki olis kiva, että nää stepit sais suoritettua tuon skriptin kautta, jolloin ei tarttis ylläpitää samaa asiaa monessa paikkaa.